packages/istanbul-reports/lib/text/index.js

```diff
@@ -66,6 +87,22 @@ function depthFor(node) {
     return ret;
 }
 
+function findMissingWidth(node, context) {
+    let last = 0;
+    function compareWidth(node) {
+        const idealWidth = nodeMissing(node).length;
```

Comment:
`last = Math.max(last, nodeMissing(node).length)`?

Reply:
I can change that, but current code is more performant since only does the assigment if the condition is meet... I can do it if you want.

Reply:
Assigning `last` is insignificant, especially compared to calling nodeMissing.  I'd say lets make the change.

Reply:
Ok, I'll do it then.

I've review it and both `findNameWidth()` and `findMissingWidth()` has almost the same code, that makes sense since I used the former one as basis for the second one... Should I unify them too?

Reply:
(the same, I didn't want to touch code of others if was not neccesary)